pkg/bmc/virtualmedia_validation.go

```diff
@@ -0,0 +1,56 @@
+package bmc
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stmcginnis/gofish"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+// ValidateVirtualMedia returns nil when the BMC has support
+// for VirtualMedia and error when the BMC does not have support.
+func ValidateVirtualMedia(address string, user string, password string, insecure bool) error {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: address,
+		Username: user,
+		Password: password,
+		Insecure: insecure,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
```

Comment:
@iurygregory it would be nice if this new method could be covered by some tests. We recently added a [MockServer](https://github.com/metal3-io/baremetal-operator/blob/3f79aae54bcead50cee67f8f2b19ecd605e457f2/pkg/provisioner/ironic/testserver/server.go#L15) to mock Ironic services, and I think it could be reused for adding some test. Probably it will require some update, as the Redfish connections requires TLS, but I saw that the `gofish.ClientConfig` accepts also an `http.Client`, so it looks feasible.